recipes/legend-pydataobj/recipe.yaml

```diff
@@ -0,0 +1,74 @@
+schema_version: 1
+
+context:
+  name: legend-pydataobj
+  version: "1.14.4"
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/legend-pydataobj-${{ version }}.tar.gz
+  sha256: 4511f5e9fce1229825c098b498148c06816f8c903338129556a0df99e4d542ef
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - setuptools >=61.2
+    - setuptools-scm >=7
+  run:
+    - python >=${{ python_min }}
+    - awkward >=2
+    - awkward-pandas
+    - colorlog
+    - h5py >=3.10
+    - hdf5plugin
+    - hist
+    - numba !=0.53.*,!=0.54.*
+    - numexpr
+    - numpy >=1.21
+    - pandas >=1.4.4
+    - parse
+    - pint >0.24
+    - pint-pandas
+
+tests:
+  - python:
+      imports:
+        - lgdo
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  - requirements:
+      run:
+        - python ${{ python_min }}.*
+        - pip
+    script:
+      - pip list
+      - lh5ls --help
+      - lh5concat --help
+
+about:
+  license: GPL-3.0
+  license_file: LICENSE
+  summary: LEGEND Python Data Objects
+  description: |
+    This package provides a Python implementation of the LEGEND Data Objects
+    (LGDO) and I/O to HDF5, including Numba-accelerated custom compression
+    algorithms for particle detector signals.
+    More documentation is available in the
+    [LEGEND data format specification](https://legend-exp.github.io/legend-data-format-specs).
+  homepage: https://github.com/legend-exp/legend-pydataobj
+  repository: https://github.com/legend-exp/legend-pydataobj
+  documentation: https://legend-pydataobj.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - matthewfeickert
```

Comment:
@gipert I'm adding this following a discussion at PyHEP.dev 2025 to move towards getting https://github.com/legend-exp/pygama on conda-forge. If you would like to be listed as a maintainer that would be excellent, but it is fine if you don't want to be.